mingw-w64-highlight/PKGBUILD

```
@@ -0,0 +1,26 @@
+_realname=highlight
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=4.3
+pkgrel=1
+url="http://www.andre-simon.de/doku/highlight/highlight.html"
+pkgdesc="Fast and flexible source code highlighter (CLI version)"
+license=('GPL')
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-lua"
+             "${MINGW_PACKAGE_PREFIX}-boost")
+source=("https://gitlab.com/saalen/highlight/-/archive/v${pkgver}/highlight-v${pkgver}.tar.bz2")
+sha256sums=('3377562676ce1aadd97648aa0c4e4e85d26db190a090df49587ae8d2471ca851')
+options=('strip')
+
+build() {
+  cd "$srcdir/${_realname}-v${pkgver}/"
+  make
```

No file was installed!
```
  [MSYS2 CI] mingw-w64-highlight: File listing diff for mingw-w64-x86_64-highlight
  
  warning: database file for 'ci' does not exist (use '-Fy' to download)
```
I think you should change it to:
```suggestion
  make PREFIX=${MINGW_PREFIX}
```